src/data/ACTIONS/root/SAM.ts

```diff
@@ -25,14 +19,28 @@ export const SAM = ensureActions({
 		},
 	},
 
+	GYOFU: {
+		id: 36963,
```

Comment:
Need to add this to the combo steps for Yukikaze too, it's throwing tons of broken combo issues